app/views/admin/people/_form.html.slim

```diff
@@ -0,0 +1,125 @@
+= simple_form_for(@person,
+    url: @form_action_url,
```

Comment:
I just realized this shouldn't be necessary. I think if you pass `[:admin, @person]` as the first arg, it should work. Then you can strip this out. If I'm right, please strip out from all of the controllers/forms.